lib/openstax/rescue_from/wrapped_exception.rb

```diff
@@ -0,0 +1,76 @@
+module OpenStax
+  module RescueFrom
+    class WrappedException
+      attr_reader :exception
+
+      def initialize(exception)
+        @exception = exception
+      end
+
+      def header
+        @header ||= cause.blank? ? 'An exception occurred' : 'Exception cause'
```

Comment:
This logic seems reversed?  cause is only going to be blank for the first cause, but "Exception cause" will print for all of the others.   Maybe OBE based on other comments.